File: WireGuardClientLibAndroid/app/src/main/java/com/example/wireguardclientlib/WireGuardStats.java
package com.example.wireguardclientlib;

public class WireGuardStats {
    public long uploadedBytes;
    public long downloadedBytes;

    public WireGuardStats(long uploadedBytes, long downloadedBytes) {
        this.uploadedBytes = uploadedBytes;
        this.downloadedBytes = downloadedBytes;
    }
}

File: WireGuardClientLib/Sources/WireGuardClientLib/WireGuardStats.swift
import Foundation

public struct WireGuardStats {
    public let uploadedBytes: UInt64
    public let downloadedBytes: UInt64

    public init(uploadedBytes: UInt64, downloadedBytes: UInt64) {
        self.uploadedBytes = uploadedBytes
        self.downloadedBytes = downloadedBytes
    }
}
